src/libs/ajax/User.ts

```diff
@@ -162,20 +202,7 @@ export const User = (signal?: AbortSignal) => {
       },
 
       // We are not calling Thurloe directly because free credits logic was in orchestration
```

Comment:
totally don't have to, but you could remove this comment while we're here - free credits hasn't been a thing for a while, you and I know that, but I could see someone newer who might get confused.